packages/wonder-blocks-tooltip/src/components/tooltip.tsx

```diff
@@ -102,6 +102,14 @@ type Props = AriaProps &
          * Test ID used for e2e testing.
          */
         testId?: string;
+        /**
+         * Optional custom styles for the tooltip content which are a subset of valid CSS styles.
+         */
+        contentStyle?: ContentStyle;
+        /**
+         * Optional background color.
+         */
+        backgroundColor?: keyof typeof Color;
```

Comment:
content style contains subsets of CSS styles but here background color is more restrictive than the CSS background color prop. i thought it would be more appropriate to factor it out! 

in the code, it gets passed down in two places (i think to `TooltipContent` and `TooltipTail`) and i felt it was more clear if it was a separate prop. open to changing it if you think it would be fine inside `contentStyle` tho!